operator_ui/src/pages/Jobs/utils.ts

```diff
@@ -111,6 +114,22 @@ export function getTaskList({
         observationSource.some((node) => !node.parentIds.length)
           ? observationSource
           : false
+      if (list) {
+        list.every((listItem) => {
+          const obj = _countBy(listItem.parentIds)
+          Object.entries(obj).every(([parentId, value]) => {
+            if (value > 1) {
+              error += `${parentId} has duplicate ${listItem.id} children`
```

Comment:
This seems to only handle this specific error. What happens if a different type of DOT error occurs?

Reply:
That's correct. It handles only the error that was mentioned in the attached ticket. This error is currently crashing the UI and you are not able to access the page until you clear local storage.
If there is a list of potential errors I would happily add their handling.

Reply:
If there is an error on job creation it will be handled by the response.